src/Microsoft.DotNet.Cli.Utils/Extensions/LockFileExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.DotNet.Tools.Common;
+using NuGet.Packaging;
+using NuGet.ProjectModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    static class LockFileExtensions
+    {
+        public static string GetPackageDirectory(this LockFile lockFile, LockFileTargetLibrary library)
+        {
+            var packageFolders = lockFile.GetNormalizedPackageFolders();
+
+            var packageFoldersCount = packageFolders.Count();
+            var userPackageFolder = packageFoldersCount == 1 ? string.Empty : packageFolders.First();
```

Comment:
If there was just one packages folder listed then it looks like the user packages folder would be skipped, possibly this works because the CLI always adds in the fallback folder now so this value is always two or higher.